pkgs/development/python-modules/phik/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pytest-pylint
+, nbconvert
+, jupyter_client
+, numpy
+, scipy
+, pandas
+, matplotlib
+, numba
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+  format = "wheel";
+
```

generally, we would declare that it's disabled here, and remove the disableIf in top-level/python-modules.nix
```suggestion
  disabled = !isPy3k;
```